openapi/eyca.yaml

```diff
@@ -158,7 +158,53 @@ paths:
           description: Service unavailable.
           schema:
             $ref: "#/definitions/CommonEycaResponse"
-  
+  "/delete_card":
```

Comment:
Fix the base path above this. It should be something like /api/v1/eyca not just /api.

Reply:
These are 3rd parties apis so we must let them this way :)